ProfessorAssessor/View Controllers/HomeViewController.m

```diff
@@ -16,14 +17,22 @@ - (void)viewDidLoad {
 }
 
 - (IBAction)logout:(UIBarButtonItem *)sender {
-    [PFUser logOutInBackgroundWithBlock:^(NSError * _Nullable error) {
-        UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
-        LoginViewController *loginViewController = [storyboard instantiateViewControllerWithIdentifier:@"LoginViewController"];
+    [PFUser logOutInBackgroundWithBlock:^(NSError *_Nullable error) {
+        if (FBSDKAccessToken.currentAccessTokenIsActive) {
+            [[FBSDKLoginManager alloc] logOut];
+        }
 
-        SceneDelegate *sceneDelegate = (SceneDelegate *)self.view.window.windowScene.delegate;
-        
-        [sceneDelegate changeRootViewController:loginViewController];
+        [self displayLoginPage];
     }];
 }
 
+- (void)displayLoginPage {
```

Comment:
Love to see it. So easy to read the method above and know exactly what it does! :D